migration/src/main/resources/migration/001_migration_7x10_to_11/040_MigrateConfigResources.yml

```
@@ -1,2 +1,3 @@
 type: specs.intershop.com/v1beta/migrate
 migrator: com.intershop.customization.migration.pfconfigurationfs.MigrateConfigResources
+message: "feat: configuration resources '*.resource' files into '*.properties'"
```

IMHO the type should be "refactor"
```suggestion
message: "refactor: convert configuration resources '*.resource' files to '*.properties'"
```